index.html

```diff
@@ -1,22 +1,23 @@
 <!doctype html>
 <!--
-Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
-The complete set of authors may be found at http://polymer.github.io/AUTHORS
-The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+  Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+  This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+  The complete set of authors may be found at http://polymer.github.io/AUTHORS
+  The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+  Code distributed by Google as part of the polymer project is also
+  subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
 -->
 <html>
 <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
   <script src="../webcomponentsjs/webcomponents.js"></script>
-  <link rel="import" href="../core-component-page/core-component-page.html">
-
+  <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
 </head>
-<body unresolved>
+<body>
 
-  <core-component-page></core-component-page>
+  <iron-doc-viewer src="iron-range.html"></iron-doc-viewer>
```

Comment:
I think that it's safe to use `<iron-component-page>` for this now :construction_worker: